packages/sdk/src/block/block.js

```diff
@@ -0,0 +1,21 @@
+import {send} from "../send/send.js"
+import {getBlock} from "../build/build-get-block"
+import {decodeResponse as decode} from "../decode/decode.js"
+import {invariant} from "@onflow/util-invariant"
+import {atBlockHeight, atBlockId} from "../sdk"
+
+export function block({sealed = false, id, height} = {}) {
```

Comment:
would `latestSealed` make this clearer? `sdk.block({latestSealed: true})`

Reply:
you can get a previous block that has been sealed (by id, etc), but this is checking for the latest specifically, right?